qdrant-landing/content/documentation/hybrid-cloud/hybrid-cloud-cluster-creation.md

```diff
@@ -155,4 +167,20 @@ metadata:
 type: kubernetes.io/tls
 ```
 
-With this command the secret name to enter into the UI would be `qdrant-tls` and the keys would be `tls.crt` and `tls.key`.
\ No newline at end of file
+With this command the secret name to enter into the UI would be `qdrant-tls` and the keys would be `tls.crt` and `tls.key`.
+
+### Configuring CPU and memory resource reservations
+
+When creating a Qdrant database cluster, Qdrant Cloud schedules Pods with specific CPU and memory requests and limits to ensure optimal performance. It’s important to use equal requests and limits for stability. Ideally, Kubernetes nodes should match the Pod size, with one database Pod per VM. 
```

Comment:
@bashofmann I tried to simplify this section. Can you please check?

Reply:
I did modify it slightly, but otherwise great rewrite, Thanks.